jupyterhub/values.yaml

```diff
@@ -102,6 +102,9 @@ auth:
     users:
   dummy:
     password:
+  state:
+    enable: false
```

Comment:
we use 'enabled' elsewhere. Use that here too to be consistent?

Reply:
Okay!

Reply:
Updated! 🙂